azure_functions_worker/constants.py

```diff
@@ -11,8 +11,10 @@
 RPC_HTTP_TRIGGER_METADATA_REMOVED = "RpcHttpTriggerMetadataRemoved"
 WORKER_STATUS = "WorkerStatus"
 
-# Debug Flags
+# Debug Flags and Worker Test Util Configuration
 PYAZURE_WEBHOST_DEBUG = "PYAZURE_WEBHOST_DEBUG"
+PYAZURE_WORKER_DIR = "PYAZURE_WORKER_DIR"
```

Comment:
Do you envision a new flag coming in for Test specific configurations in the future? If yes, then please split this group of variables into two.